src/main/java/com/google/codeu/data/User.java

```diff
@@ -1,13 +1,24 @@
 package com.google.codeu.data;
 
+//Provides the capability to save the data the user submits, 
+//and then retrieve that data to display the user's "about me" 
+//section on their user page.
+
 public class User {
 
     private String email;
     private String aboutMe;
 
+    //If null is passed in through the method then return empty string for each field. 
     public User(String email, String aboutMe) {
+      if(email == null || aboutMe == null){
```

Comment:
This would blank out the email and user if either the aboutMe or email is null.

If you want to set the values to empty string if the value is null, you probably want to use [Objects#toString](https://docs.oracle.com/javase/7/docs/api/java/util/Objects.html#toString(java.lang.Object,%20java.lang.String))
for each of the values being set.